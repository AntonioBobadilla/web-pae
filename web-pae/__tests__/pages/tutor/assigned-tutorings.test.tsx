import AssignedTutorings from '@/pages/tutor/assigned-tutorings';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Assigned Tutorings', () => {
  it('renders assigned tutorings page unchanged', () => {
    const { container } = render(<AssignedTutorings />);
    expect(container).toMatchSnapshot();
  });
});
