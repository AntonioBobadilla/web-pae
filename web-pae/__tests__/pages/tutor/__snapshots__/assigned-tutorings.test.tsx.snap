// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Assigned Tutorings renders assigned tutorings page unchanged 1`] = `
<div>
  <h1>
    ASESOR√çAS ASIGNADAS
  </h1>
</div>
`;
